yajsapi/executor/executor.ts

```diff
@@ -268,7 +265,7 @@ export class TaskExecutor {
    * @example
    * ```typescript
    * const data = [1, 2, 3, 4, 5];
-   * const results = executor.map(data, (ctx, item) => providerCtx.ctx(`echo "${item}"`));
+   * const results = executor.map(data, async (ctx, item) => ctx.run(`echo "${item}"`));
```

Comment:
There's no use in `async` if you don't `await` anything in the body. of the function.